wire-ios-data-model/Source/MLS/SafeCoreCrypto.swift

```diff
@@ -116,18 +133,36 @@ public class SafeCoreCrypto: SafeCoreCryptoProtocol {
         return result
     }
 
-    // TODO: Remove when async core crypto is supported
-    public func perform<T>(_ block: (WireCoreCrypto.CoreCryptoProtocol) async throws -> T) async rethrows -> T {
-        try await block(coreCrypto)
+    public func perform<T>(_ block: (WireCoreCrypto.CoreCryptoProtocol) throws -> T) async rethrows -> T {
```

Comment:
This method can be removed, right?
It won't be an issue if a provided block contains no `await` and is called with
```
try await block()
```

So the `perform` method above should be sufficient.

Reply:
I think you're right, let's remove it.

Reply:
Done